frontend/src/models/index.js

```diff
@@ -252,7 +261,14 @@ export const RootModel = types
           };
 
           const result = yield postReservation(payload);
-          console.log(result);
+
+          payload.tickets.forEach(
```

Comment:
How about putting this into a function, since you are doing it twice here and now the code is copy-pasted in two places?

Reply:
yes, i will write a function to do the job